pkg/el/google-fluentd-catch-all-config.spec

```diff
@@ -0,0 +1,40 @@
+Summary: Catch-all configuration files for google-fluentd on GCE.
+Name: google-fluentd-catch-all-config
+Version: 0.1
+Release: 1%{?dist}
+License: Apache
+Group: System Environment/Daemons
+URL: https://cloud.google.com/logging/
+Requires: google-fluentd
+
+%description
+A set of configuration files for google-fluentd used to ingest logs
+from the system and third-party application packages.
+
+%files
+%config /etc/google-fluentd
+
+%clean
+# don't clean up the files here; we'll do it in the toplevel Makefile
```

Comment:
When I used the default one, it was cleaning up things I didn't want it to, which is why I did this... but I will try it again with buildroot.